README.md

```diff
@@ -8,7 +8,7 @@
 
 
 Mypy stubs for [DRF 3.12.x](https://pypi.org/project/djangorestframework/).
-Supports Python 3.6, 3.7 and 3.8.
+Supports Python 3.6, 3.7, 3.8 and 3.9.
```

Comment:
Maybe just Python 3.6+ ?

Reply:
I would rather suggest Python 3.6-3.9. Anyway there is nothing wrong with explicitly listing versions (see DRF's [readme](https://github.com/encode/django-rest-framework#requirements))